ppr-ui/src/views/userAccess/QsInformation.vue

```diff
@@ -14,57 +14,92 @@
       />
     </section>
 
-    <section class="qs-service-agreement mt-8">
-      <h2>Service Agreement</h2>
-      <p class="mt-1">
-        Read and accept the terms of the service agreement. You may download and a copy of this agreement to save for
-        your records.
-      </p>
-      <v-btn outlined color="primary" class="mt-2" :ripple="false">
-        <img alt="" src="@/assets/svgs/pdf-icon-blue.svg" />
-        <span class="pl-1">Download Qualified Suppliers Agreement</span>
-      </v-btn>
-    </section>
+    <div class="increment-sections">
+      <section class="qs-service-agreement mt-8">
+        <h2>Service Agreement</h2>
+        <p class="mt-1">
+          Read and accept the terms of the service agreement. You may download and a copy of this agreement to save for
+          your records.
+        </p>
 
-    <section class="qs-information-form mt-8">
-      <h2>Qualified Supplier ({{ getMhrSubProduct }}) Information</h2>
-      <p class="mt-1">
-        Provide the name and contact information for the Qualified Supplier.
-      </p>
+        <!-- download service agreement button -->
```

Comment:
Would be a good candidate for common Download btn component

Reply:
Sure, can do!

Reply:
All good, already done!